src/site/_includes/components/Meta.js

```diff
@@ -106,10 +106,21 @@ module.exports = (locale, page, collections, renderData = {}) => {
 
   function renderTwitterMeta() {
     const meta = getMetaByPlatform('twitter');
+    /**
+     * We replace the `<` and `>` characters for Twitter because HTML tags
```

Comment:
Maybe explicitly state that Twitter also strips out `&lt;` and `&gt;` since I think most folks would normally reach for those.